react-native-libraries.json

```diff
@@ -1736,11 +1736,6 @@
     "githubUrl": "https://github.com/cesardeazevedo/react-native-bottom-sheet-behavior",
     "android": true
   },
-  {
-    "githubUrl": "https://github.com/tinycreative/react-native-intercom",
-    "ios": true,
-    "android": true
-  },
```

Comment:
If the package exists in npm registry we usually mark it as deprecated, instead of removing it, since it can be a good source of information for people to migrate off. The information for the directory API is also used by different tools, like `expo-doctor` to warn about and aid the migration.

We also support package suggestion, so in this case it would be:
```json
  {
    "githubUrl": "https://github.com/tinycreative/react-native-intercom",
    "ios": true,
    "android": true,
    "unmaintained": "@intercom/intercom-react-native"
  },
```

Reply:
Great insight thanks, I made the update 🙂